utils.py

```diff
@@ -16,7 +16,7 @@
 
 from loguru import logger
 
-def get_class_distribution(label, display, save, title):
+def get_class_distribution(label, display, save = False, title = None):
```

Comment:
```suggestion
def get_class_distribution(label, display, save=False, title=None):
```
PEP8

Reply:
Thanks, I will review the entire code to match to PEP8